packages/arcs/tests/boundingBox.test.ts

```diff
@@ -0,0 +1,52 @@
+import { computeArcBoundingBox } from '../src/boundingBox'
+
+const boundingBoxCases = [
+    {
+        args: [0, 0, 100, 0, 360],
+        expected: {
+            x: -100,
+            y: -100,
+            width: 200,
+            height: 200,
+        },
+    },
+    {
+        args: [0, 0, 100, 0, 90],
+        expected: {
+            x: 0,
+            y: 0,
+            width: 100,
+            height: 100,
+        },
+    },
+    {
+        args: [0, 0, 100, -90, 0],
+        expected: {
+            x: 0,
+            y: -100,
+            width: 100,
+            height: 100,
+        },
+    },
+    {
+        args: [0, 0, 100, 90, 180],
+        expected: {
+            x: -100,
+            y: 0,
+            width: 100,
+            height: 100,
+        },
+    },
+]
+
+for (const boundingBoxCase of boundingBoxCases) {
+    const { args, expected } = boundingBoxCase
+
+    test(`computeArcBoundingBox() for position ${args[0]}, ${args[1]} with radius ${args[2]}, starting at ${args[3]}°, ending at ${args[4]}°`, () => {
```

Comment:
There is also `test.each`, but this works too.

Reply:
Thank you, I didn't know! I think we can keep it like this, apparently it accepts an array of arrays, I'm not sure it's worth changing it.